tracer/src/Datadog.Tracer.Native/iast/dataflow.cpp

```diff
@@ -188,6 +188,7 @@ void Dataflow::Destroy()
     if (_initialized)
     {
         _initialized = false;
+        _loaded = false;
```

Comment:
I'm a bit confused regarding the difference between loaded and initialized?  maybe this is `_aspectsLoaded`?

Reply:
Yeah, that's mainly it. Poorly chosen names